.vscode/settings.json

```diff
@@ -5,7 +5,7 @@
   "editor.defaultFormatter": "esbenp.prettier-vscode",
   "editor.formatOnSave": true,
   "eslint.rules.customizations": [{ "rule": "*", "severity": "warn" }],
-  "explorer.excludeGitIgnore": true,
+  // "explorer.excludeGitIgnore": true,
```

Comment:
Can we just remove this permanently

Reply:
lol

![image](https://github.com/pingdotgg/uploadthing/assets/11494384/cea34e82-fddc-4d9b-8ec3-122f10c080ee)

Reply:
Haha Yea we can keep it off